docs/source/get-started/suggested-models.md

```diff
@@ -0,0 +1,173 @@
+# Suggested Models
+
+In theory, Lumigator supports any model that is uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
+and is trained for *summarization*. However, we have extensively tested a few models and created
+an endpoint to easily retrieve them.
+
+In this guide, we assume that you have already [installed Lumigator locally](quickstart), and have a
+running instance. To get a list of suggested models, you can use the following command:
+
+::::{tab-set}
+
+:::{tab-item} cURL
+:sync: tab1
+
+```console
+user@host:~/lumigator$ curl -s http://localhost:8000/api/v1/models/summarization | jq
+{
+  "total": 9,
+  "items": [
+    {
+      "name": "facebook/bart-large-cnn",
+      "uri": "hf://facebook/bart-large-cnn",
+      "description": "BART is a large-sized model fine-tuned on the CNN Daily Mail dataset.",
+      "info": {
+        "parameters_count": "406M",
+        "tensor_type": "F32",
+        "model_size": "1.63GB"
+      },
+      "tasks": [
+        {
+          "summarization": {
+...
+```
+
+:::
+
+:::{tab-item} Python SDK
+:sync: tab2
+```python
+from lumigator_sdk.lumigator import LumigatorClient
+
+lm_client = LumigatorClient("localhost:8000")
```

Comment:
suggestion: leave a comment that the default port is 8000 for lumigator so folks know?

Reply:
I also added an admonition.